### PR TITLE
chore: Upload ccp snapshots to s3

### PR DESCRIPTION
**Pull request by nahsi**

### CODE REVIEW

File: .github/workflows/artifact.yml [hunk 1]
```diff
@@ -22,6 +22,10 @@ jobs:
       RUSTFLAGS: "-D warnings -C link-arg=-B/usr/local/bin/mold"
       RUSTC_WRAPPER: "sccache"
 
+    permissions:
```
Comment (folex) [inline .github/workflows/artifact.yml:26]: comment

File: .github/workflows/artifact.yml [hunk 2]
```diff
@@ -72,6 +76,30 @@ jobs:
...
+          method: jwt
+          jwtGithubAudience: "https://github.com/fluencelabs"
+          jwtTtl: 300
+          secrets: |
+            kv/ci/fluence-artifacts id | AWS_ACCESS_KEY_ID ;
+            kv/ci/fluence-artifacts secret | AWS_SECRET_ACCESS_KEY
+
+      - name: Rename ccp binary
+        run: |
+          mkdir s3
```
Comment (folex) [inline .github/workflows/artifact.yml]:
```suggestion
          mkdir -p s3
```

File: .github/workflows/artifact.yml [hunk 3]
```diff
@@ -72,6 +76,30 @@ jobs:
...
+          cp target/${{ matrix.target }}/release/ccp s3/ccp-${{ matrix.arch }}
```
Comment (folex) [inline .github/workflows/artifact.yml:96]:
```suggestion
          cp "target/${{ matrix.target }}/release/ccp" "s3/ccp-${{ matrix.arch }}"
```

**Review by folex — APPROVED** [review]
Can you make it print link to S3 in the CI run Summary?

File: .github/workflows/artifact.yml (hunk 1 above)
Comment (nahsi) [inline .github/workflows/artifact.yml:26]:
```suggestion
    # required for vault-action
    permissions:
```

**Pull request merged**